k8s/namespaces/rd/rd-location-ref-data-load-integration/rd-location-ref-data-load-integration.yaml

```diff
@@ -14,4 +14,4 @@ spec:
   values:
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/location-ref-data-load:prod-359a54a-20210831113321
+      image: hmctspublic.azurecr.io/rd/location-ref-data-load:prod-359a54a-20210831113321   #{"$imagepolicy": "flux-system:rd-location-ref-data-load-integration"}
```

Comment:
Can you change this policy to `rd-location-ref-data-load`? It looks to be using the same, I would remove the image repo/policy for `rd-location-ref-data-load`

Reply:
Image policy/repo now updated.